package.json

```
@@ -23,7 +23,8 @@
     "dev:env": "vite dev-env --config ./vite.config.js",
     "test": "vitest",
     "build": "rollup -c",
-    "version": "pnpm test && pnpm build && pnpm publish"
+    "version": "pnpm test && pnpm build && pnpm publish",
+    "lint": "eslint . --fix"
```

What do you think about this suggestion? Just to be clear that you are not only running lint but also fixing the issues? I do not have a strong opinion here though.

```suggestion
    "lint:fix": "eslint . --fix"
```